src/index.js

```diff
@@ -48,4 +48,6 @@ var namespaces         = 0
 	return getter;
 }
 
-module.exports = { getNamespace, getCountNamespaces, startRequest, bind, patch };
+module.exports = global.requestLocalStorage || { getNamespace, getCountNamespaces, startRequest, bind, patch };
+
+if (!global.requestLocalStorage) global.requestLocalStorage = module.exports;
```

Comment:
Does that address the issue @karlhorky is having?

I had thought he was in a situation where two packages each had their own copy of RLS.